deploy/roles/webserver/tasks/main.yml

```diff
@@ -156,6 +156,11 @@
   sudo_user: mindleaps
   shell: DEVISE_SECRET_KEY={{ devise_secret_key }} RAILS_ENV=production ~/.rvm/bin/rvm `cat ~/tracker/.ruby-version` do rails db:migrate chdir=/home/mindleaps/tracker
 
+- name: Reset Postgres sequences of table primary key ids
+  sudo: true
+  sudo_user: mindleaps
+  shell: DEVISE_SECRET_KEY={{ devise_secret_key }} RAILS_ENV=production ~/.rvm/bin/rvm `cat ~/tracker/.ruby-version` do rake db:correct_sequence_ids chdir=/home/mindleaps/tracker
```

Comment:
Yeah, I'm not sure. On one hand there are no negatives of running this always. On the other, it's one more deployment step that makes it a bit more complicated.

What do you think? Should we keep it just as a rake task and run when needed or keep it in the deploy script?